django/webscraper/models.py

```diff
@@ -0,0 +1,77 @@
+"""This module defines models (and useful functions connected to them),
+for webscraper application"""
+import datetime
+from django.db import models
+
+
+# model was previously in pieskiUW.models, but models shouldn't
```

Comment:
This comment isn't needed in my opinion. It doesn't matter where it was previously if it is needed here now.

Reply:
This conversation was marked as resolved, but this comment should be deleted. It clutters the code and add unnecessary information about changes in the repository. This file was in another location due to the initial state of this project. You moving it is very much appreciated.